### ISSUE TITLE
Bug: Error message should display utf8 correctly

### ISSUE THREAD

**Issue by 1c7** [ClickHouse/clickhouse-connect#18]
# Reference
https://github.com/dbt-labs/dbt-core/issues/5375#issuecomment-1156109300

# Expect behavior
Error Message show Chinese charactor as is. 

# What actually happen
I got something like `\xe5\xa5\xbd`

# How to Reproduce Error
```python
import clickhouse_connect

client = clickhouse_connect.get_client(
  host='ec2-xx-131-225-xxx.cn-north-1.compute.amazonaws.com.cn', 
  port=8123, 
  username='default',
  password="[hide]",
  database='mp_production',
)

query_result = client.query("""
  SELECT * FROM apply_applications where id = '好'
""")

print (query_result.result_set) 
```
The SQL here is wrong, but the point is error message should display `好`, not `\xe5\xa5\xbd`

# Screenshot
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/1804755/174249124-82a4e3be-3b31-462d-9f23-702f261c17b8.png">

**Comment by 1c7** [comment]
Thank you for maintain this package!

**Comment by genzgd** [comment]
This should be fixed in just released 0.1.3
